.golangci.yml

```diff
@@ -12,7 +12,6 @@ linters:
   enable:
     - deadcode
     - gci
-    - godot
```

Comment:
I removed the **godot** linter @javorszky. It's a bit too strict and unhelpful if there is a line with code (ending in say `}`.

Reply:
Fair point